tpm_core_node:
  ros__parameters:
    robot_type: 21
    a: [0.0,170.0,270.0,170.0,0.0,0.0,0.0]
    alpha: [0.0,0.0,0.0,0.0,0.0,0.0,0.0]
    d: [320.0,-31.0,-39.0,-47.0,0.0,0.0,0.0]
    theta: [0.0,0.0,0.0,0.0,0.0,0.0,0.0]
    theta_shift: [0.0,0.0,0.0,0.0,0.0,0.0,0.0]
    pos_limit: [500.0,90.0,90.0,90.0,90.0,90.0,0.0]
    neg_limit: [-10.0,-90.0,-90.0,-90.0,-90.0,-90.0,0.0]
    pulse_per_unit: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]
    max_xyz_jog_speed: 40.0
    max_abc_jog_speed: 20.0
    